web/src/Root.tsx

```diff
@@ -608,7 +608,7 @@ const Root = () => {
                   )}
                 />
                 <ProtectedRoute
-                  path={["/snapshots/:tab?"]}
+                  path={["/snapshots/:tab?/:slug?"]}
```

Comment:
this slug is already part of the url but we don't pass it into react router as a variable. this slug is needed to know what the selected app is in the snapshots page.

snapshots are a root page so there's no selected app in the way you have one on the dashboard page. it's selected from a drop down.